src/components/Form/Checkbox.js

```diff
@@ -25,15 +25,15 @@ const Checkbox = ({
     : disabled
     ? colorScheme.set('borderColor', 'disabled')
     : black
-    ? colorScheme.set('borderColor', '#000')
+    ? colorScheme.set('borderColor', 'logo')
```

Comment:
just out of curiosity: where is Checkbox still used with `black` that lead to this change? We started to get ride of that in `Field`. And should probably continue soon with checkbox (not as part of this PR).

Reply:
ah that was in a legacy page `/umfrage/1-minute` – I thought it might come up again in the future so I decided to fix it. But if you are telling me this is soon to be deprecated, I can revert that change. Should I?

Reply:
thx. leave it as is, just wanted to know the culprit page